app/ui/mainwindow2.ui

```diff
@@ -632,34 +632,26 @@
    </property>
    <property name="icon">
     <iconset resource="../data/app.qrc">
-     <normaloff>:/icons/controls/loop.png</normaloff>:/icons/controls/loop.png</iconset>
+     <normaloff>:/icons/themes/playful/controls/control-loop.svg</normaloff>:/icons/themes/playful/controls/control-loop.svg</iconset>
    </property>
    <property name="text">
     <string>Loop</string>
    </property>
   </action>
   <action name="actionNext_Frame">
-   <property name="icon">
-    <iconset resource="../data/app.qrc">
-     <normaloff>:/icons/next.png</normaloff>:/icons/next.png</iconset>
-   </property>
    <property name="text">
     <string>Next Frame</string>
    </property>
   </action>
   <action name="actionPrevious_Frame">
-   <property name="icon">
-    <iconset resource="../data/app.qrc">
-     <normaloff>:/icons/prev.png</normaloff>:/icons/prev.png</iconset>
-   </property>
```

Comment:
Yes, both the prev, next and exit icons are left out on purpose.
There's something to be said about having icons in the top menu in general. Since it's never been on all icons, I'd been thinking that maybe they shouldn't be there at all. There are different requirements to icons in the top menu too, like specific sizes that the icons must use in order to look proper.